src/bindings/Node.ml

```diff
@@ -201,12 +200,8 @@ module ChildProcess = struct
     Promise.make @@ fun ~resolve ~reject:_ ->
     let cp = ref [%bs.obj { exitCode = 0 }] in
     cp :=
-      exec cmd options (fun err stdout stderr ->
-          if Js.Nullable.isNullable err then (
```

Comment:
This error checking is redundant since `Cmd.output` also checks if there is an error and provides more information.